app/services/user_facing_state.rb

```diff
@@ -37,6 +41,10 @@ def to_s
       "draft"
     elsif review_state == "published_without_review"
       "published_but_needs_2i"
+    elsif publication_state == "sent_to_live" && live_state == "retired"
```

Comment:
This might want to go above the previous clause, as I think if you retire/remove an unreviewed document it'll still come out as 'published_but_needs_2i'.

Reply:
Done. Will fix-up.